modules/image-separator/includes/frontend.js.php

```diff
@@ -12,11 +12,9 @@
 	$(document).ready(function() {
 
 		new UABBAnimation({
-			id: '<?php echo $id; ?>',
-			animation_delay: '<?php echo ( '' != $settings->img_animation_delay && '0' != $settings->img_animation_delay ) ? $settings->img_animation_delay : ''; ?>',
-			animation: '<?php echo ( 'no' == $settings->img_animation ) ? 'no' : 'animated ' . $settings->img_animation; ?>',
-			viewport_position: '<?php echo ( '' != $settings->img_viewport_position ) ? $settings->img_viewport_position : '90'; ?>'
```

Comment:
Revert this change if not intentionally done.

Reply:
Reverted. Done